src/mmw/js/src/draw/utils.js

```diff
@@ -4,10 +4,11 @@ var $ = require('jquery'),
     L = require('leaflet'),
     _ = require('lodash'),
     turfArea = require('turf-area'),
+    turfBboxPolygon = require('turf-bbox-polygon'),
     coreUtils = require('../core/utils');
 
 // Keep in sync with src/api/main.py in rapid-watershed-delineation.
-var MAX_AREA = 112700; // About the size of a large state (in km^2)
+var MAX_AREA = 75000; // About the size of a large state (in km^2)
```

Comment:
✏️  The average US state is 183,637 km². Consider changing this to "About the size of West Virginia"